.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+# Type of change
+
+Please put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# Fixes (Optional)
+
+Please list any issues that are fixed by this PR in the format `#<issue number>`. If it fixes multiple issues, please put each issue in a separate line. If this Pull Request does not fix any issues, please delete this section.
+
+# Examples/Screenshots
+
+Please attach a screenshot or screen recording of features/fixes you've implemented to verify your changes. Please also note any relevant details for your configuration. If your changes do not affect the UI, please delete this section.
+
+Use this table template to show examples of your changes:
+
+| Before       | After        |
+| ------------ | ------------ |
+| Content Cell | Content Cell |
+| Content Cell | Content Cell |
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
```

Comment:
Isn't it the responsibility of the linter to do that ?

Reply:
Correct! mb, lemme get rid of that